### DISCUSSION TITLE
Ecowitt WS90 weather station - frequency?

### DISCUSSION THREAD

**Discussion by gregmac** [merbanan/rtl_433#2896] [Q&A]
WS90 support was added in #2072 but there's a few variants:

* WS90BN North America 915Mhz
* WS90BU United Kingdom 868Mhz
* WS90BE Europe 868Mhz
* WS90BO Oceania 433Mhz 

I'm in North America, but since I have several 433Mhz sensors already I'm tempted to get that variant instead of doing rtl_433 frequency hopping. Before I buy, is there any downside to this? Does the frequency affect rtl_433's support of the protocol at all?

**Comment by zuckschwerdt** [comment]
Frequency is just a random value to rtl_433. But note that some defaults (sample rate and discriminator) are different for below / above 800M. You might want to test with both of each: `-s 250k` vs `-s 1024k` and `-Y classic` vs `-Y minmax`.
